lib/HLSL/DxilPreparePasses.cpp

```diff
@@ -657,16 +657,23 @@ class DxilFinalizeModule : public ModulePass {
           // In spite of the <=1 assert above, loop here in case the assumption is wrong
           for (auto II = U->user_begin(), EE = U->user_end(); II != EE;) {
```

Comment:
I think using CI here instead of U makes things a little clearer that we are iterating users of the CallInst.